forecast-admin/forecast/opportunities/management/commands/load_opportunities.py

```diff
@@ -0,0 +1,193 @@
+import os
+import csv
+from datetime import date
+from decimal import Decimal, InvalidOperation
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from opportunities.models import Office, Opportunity, OSBUAdvisor
+from optparse import make_option
+
+
+# A data loader of contract opportunities
+# Thank you Brendan Sudol... :)
+# https://github.com/brendansudol/calc/blob/ca4a13d0fa34a468d788710975bf71959eb8bb20/contracts/management/commands/load_s70.py
+class Command(BaseCommand):
+    help = 'Load data from CSV'
+
+    default_filename = 'forecast/fixtures/fy16q1.csv'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-f', '--filename',
+            default=default_filename,
+            help='input filename (.csv, default {})'.format(
+                default_filename
```

Comment:
I probably wouldn't do a default - so that we force people to think about what data they are uploading.